lib/mpl_toolkits/axes_grid1/axes_size.py

```diff
@@ -19,12 +19,27 @@ class _Base:
     def __rmul__(self, other):
         return Fraction(other, self)
 
+    def __mul__(self, other):
+        return Fraction(self, other)
+
+    def __div__(self, other):
+        return Fraction(self, 1/other)
+
     def __add__(self, other):
         if isinstance(other, _Base):
             return Add(self, other)
         else:
             return Add(self, Fixed(other))
 
+    def __radd__(self, other):
+        return Add(self, Fixed(other))
```

Comment:
I would add a comment, explaining the asymmetry compared to `__add__`
```suggestion
        # other cannot be a _Base instance, because A + B would trigger
        # A.__add__(B) already
        return Add(self, Fixed(other))
```